tests/specs/lemmas.k

```diff
@@ -88,118 +94,105 @@ module LEMMAS
        andBool 0 <=Int V2 andBool V2 <Int pow256
       [concrete]
 
-    rule keccakIntList(V:Int .IntList) => hash1(V)
-    rule keccakIntList(V1:Int V2:Int .IntList) => hash2(V1, V2)
+    rule keccakIntList(V:Int .IntList) => hash1(V)               [simplification]
+    rule keccakIntList(V1:Int V2:Int .IntList) => hash2(V1, V2)  [simplification]
 
     // for terms came from bytecode not via #hashedLocation
     rule keccak(WS) => keccakIntList(byteStack2IntList(WS))
-      requires ( notBool #isConcrete(WS) )
-       andBool ( #sizeByteArray(WS) ==Int 32 orBool #sizeByteArray(WS) ==Int 64 )
+      requires ( #sizeByteArray(WS) ==Int 32 orBool #sizeByteArray(WS) ==Int 64 )
+      [simplification]
```

Comment:
this change will be problematic.  we want to concretely evaluate keccak(WS) when WS is concrete.  so, i think we will need to use a rule priority to ensure that this lemma is applied only when WS is symbolic.